pybop/models/base_model.py

```diff
@@ -80,14 +80,10 @@ def __init__(
 
         Additional Attributes
         ---------------------
-        parameters : pybop.Parameters
+        pybamm_model : pybamm.BaseModel, optional
+            An instance of a PyBaMM model.
+        parameters : pybop.Parameters, optional
```

Comment:
As these are internal attributes, I think the `optional` be removed here, correct?

```suggestion
        pybamm_model : pybamm.BaseModel
            An instance of a PyBaMM model.
        parameters : pybop.Parameters
```